lib/src/upgrader.dart

```diff
@@ -239,6 +239,37 @@ class Upgrader with WidgetsBindingObserver {
         state.countryCodeOverride ?? findCountryCode(locale: locale);
     if (state.debugLogging) {
       print('upgrader: countryCode: $country');
+      // The  language code of the locale, defaulting to `en`.
```

Comment:
This code looks old and is already in the master branch. Can you check your PR merge point to see that it points to the latest code.

Reply:
> This code looks old and is already in the master branch. Can you check your PR merge point to see that it points to the latest code.

Sorry @larryaasen I would be very happy to fix the merge conflicts but since I am not working on the project that uses this package I will need more time to check how the code is working and that my changes are working well and  I am little bit busy these days with another projects so look in to the other similar PRs.